SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Supported Versions
+
+Versions of the project that are currently being supported with security updates:
+
+| Version | Supported          |
```

Comment:
I wonder if we would be better off explaining our policy (security updates come to N and N-1) than an explicit list that has to be updated with releases. Anyway, the below is fine it will just require maintenance.